### PR TITLE
Updates to generated id formats

### PR DESCRIPTION
**Pull request by slevenick**
**Release Note Template for Downstream PRs (will be copied)**

```release-note:REPLACEME

```

### CODE REVIEW

File: products/sql/terraform.yaml [hunk 1]
```diff
@@ -16,12 +16,9 @@ client_name: 'SqlAdmin'
 overrides: !ruby/object:Overrides::ResourceOverrides
   Database: !ruby/object:Overrides::Terraform::ResourceOverride
     mutex: "google-sql-database-instance-{{project}}-{{instance}}"
-    id_format: "{{instance}}:{{name}}"
     import_format: ["projects/{{project}}/instances/{{instance}}/databases/{{name}}",
                     "{{project}}/{{instance}}/{{name}}",
                     "instances/{{instance}}/databases/{{name}}",
-                    # support for a legacy import format
```
Comment (slevenick) [inline products/sql/terraform.yaml]: Do we need to note that we are no longer supporting this?

File: products/appengine/terraform.yaml [hunk 2]
```diff
@@ -14,8 +14,8 @@
 --- !ruby/object:Provider::Terraform::Config
 overrides: !ruby/object:Overrides::ResourceOverrides
   FirewallRule: !ruby/object:Overrides::Terraform::ResourceOverride
-    id_format: "{{project}}/{{priority}}"
-    import_format: ["{{project}}/{{priority}}"]
+    id_format: "apps/{{project}}/firewall/ingressRules/{{priority}}"
```
Comment (rileykarson) [inline products/appengine/terraform.yaml]: Will https://github.com/GoogleCloudPlatform/magic-modules/pull/2461 make this + some other values redundant?

File: products/sql/terraform.yaml (hunk 1 above)
Comment (rileykarson) [reply products/sql/terraform.yaml]: Probably worth adding, yeah.

**Comment by slevenick** [comment]
> Few more issues:
> 
> `{{version_id}}` replaced with `{{name}}` in google-beta/resource_app_engine_standard_app_version.go
> 
> Double entries (I suspect IAM is doing something silly for things that don't end in `name`? Idk:
> 
> * google-beta/iam_binary_authorization_attestor.go
> * google-beta/iam_runtime_config_config.go

Ah, it's that our import format default doesn't handle query params (and is based off base_url instead of self_link). It's both binauth_iam and the regular resource. I'll add back the import_id for those

**Pull request merged**